pkgs/development/tools/misc/src-cli/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "src-cli";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "sourcegraph";
+    repo = pname;
```

For readabilities sake its better to spell this out IMO
```suggestion
    repo = "src-cli";
```